omod-common/src/main/java/org/openmrs/module/webservices/helper/ModuleAction.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.helper;
+
+import org.openmrs.module.Module;
+
+import java.util.Collection;
+
+public class ModuleAction {
+	
+	public enum Type {
```

Comment:
I would call this enum Action instead of Type. What do you think @PawelGutkowski? :)